src/style.css

```
@@ -0,0 +1,25 @@
+body {
+  font-size: 13px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+h1 {
+  font-size: 17px;
+  margin: 0;
+}
+
+h2 {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 0;
+  margin-bottom: 1em;
+  margin-top: 1em;
```

```suggestion
  margin: 1em 0;
```
It means then top and bottom margins are set to 1em and left and right margins are set to 0